runtime/moonbase/src/lib.rs

```diff
@@ -911,30 +919,23 @@ impl Contains<Call> for NormalFilter {
 			// We also disable destroy, as we want to route destroy through the
 			// asset-manager, which guarantees the removal both at the EVM and
 			// substrate side of things
-			Call::LocalAssets(method) => match method {
+			RuntimeCall::LocalAssets(method) => match method {
 				pallet_assets::Call::create { .. } => false,
 				pallet_assets::Call::destroy { .. } => false,
 				_ => true,
 			},
 			// We just want to enable this in case of live chains, since the default version
 			// is populated at genesis
-			Call::PolkadotXcm(method) => match method {
+			RuntimeCall::PolkadotXcm(method) => match method {
 				pallet_xcm::Call::force_default_xcm_version { .. } => true,
 				_ => false,
 			},
-			// We filter anonymous proxy as they make "reserve" inconsistent
-			// See: https://github.com/paritytech/substrate/blob/37cca710eed3dadd4ed5364c7686608f5175cce1/frame/proxy/src/lib.rs#L270 // editorconfig-checker-disable-line
-			Call::Proxy(method) => match method {
-				pallet_proxy::Call::anonymous { .. } => false,
-				pallet_proxy::Call::kill_anonymous { .. } => false,
-				_ => true,
-			},
```

Comment:
Is this intentional?

Reply:
Good point. I initally though these methods had been removed, but instead they have been renamed to `create_pure `and `kill_pure`. I will disable those again, but maybe it is safe to incorporate them now (CC @crystalin )